src/store/folder/utils.ts

```diff
@@ -15,7 +15,7 @@ const getOriginalId = (f: Folder): string => {
 export const sortFolders = (f: Folder): string => {
 	const id = getOriginalId(f);
 	if (id === FOLDERS.TRASH) {
-		return '~';
+		return '16.1';
```

Comment:
What should this value represent? Maybe a comment and/or a constant could be clearer

Reply:
this value takes the trash folder in the last position of the system folders. I assigned a constant to it